src/fidesops/models/privacy_request.py

```diff
@@ -186,10 +186,17 @@ def cache_drp_request_body(self, drp_request_body: DrpPrivacyRequestCreate) -> N
         drp_request_body_dict: Dict[str, Any] = dict(drp_request_body)
         for key, value in drp_request_body_dict.items():
             if value is not None:
-                cache.set_with_autoexpire(
-                    get_drp_request_body_cache_key(self.id, key),
-                    value,
-                )
+                # handle nested dict/objects
+                if not isinstance(value, (bytes, str, int, float)):
```

Comment:
cache doesn't handle vals as bytes/str/int/float. Perhaps we should instead cache the whole req body as a str (`json.loads()`) instead of separating the top level props as cache keys. But this works as far as caching for debugging purposes only.